library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-ventricular-premature-depolarization <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-tachycardia <nowiki> {suggestedTag=Frequency}[Numerical value for frequency in beats/minute. (Source: Beniczky ea 2017, Table 16; Beniczky ea 2013, Table 9.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** Other-ECG-feature {requireChild} <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * EMG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording, suggestedTag=Symmetrical, suggestedTag=Left, suggestedTag=Right} [Findings in Electromyography recordings (Source: Beniczky ea 2017, Section 13, Table 16.). Suggested tags can be used to note the side of the muscle (Left or Right). The order of activation may be indicated using Temporal-relation: (Left, (Before,Right)).]</nowiki>
         ** EMG-muscle-name
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** EMG-feature
-            *** EMG-myoclonus
-                **** Negative-myoclonus
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** EMG-myoclonus-rhythmic
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-arrhythmic
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-synchronous
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-asynchronous
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-PLMS <nowiki>[Periodic limb movements in sleep.]</nowiki>
-            *** EMG-spasm
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-tonic-contraction
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-asymmetric-activation 
-              **** EMG-asymmetric-activation-left-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-              **** EMG-asymmetric-activation-right-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-EMG-features {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-polygraphic-channel {requireChild}
+        ** Myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
```

Comment:
Suggest description:
"A brief, involuntary, irregular (lacking rhythm) twitching of a muscle, a joint, or a group of muscles. (Source: Wikipedia; Beniczky ea 2017, Table 16.)"